### ISSUE TITLE
Update On-Chain Status Display for Individual Credentials in Passport Client

### ISSUE THREAD

**Issue by erichfi** [gitcoinco/passport#1366]
### User Story:
As a Gitcoin Passport holder,
I want to see the on-chain status of my individual credentials updated in the Passport client through the Resolver,
So that I can understand the current state of my digital identity in a decentralized context.

### Acceptance Criteria
GIVEN I am a Gitcoin Passport holder with one or more credentials,
WHEN I view my Passport client,
THEN I should see the on-chain status of each individual credential clearly displayed, updated according to the new on-chain Passport schema through the `GitcoinPassportResolver`.

GIVEN an update has been made to the on-chain status of my credentials via the `GitcoinPassportResolver`,
WHEN I refresh my Passport client,
THEN the displayed on-chain status of the affected credentials should reflect this update.

#### Product & Design Links:
n/a

#### Tech Details:
This update will involve aligning the Passport client's display functionality with the new on-chain schema for Passport credentials via the `GitcoinPassportResolver`. All credentials displayed should clearly indicate their on-chain status, with real-time updates as changes occur.

#### Open Questions:
- What should be the exact behavior in case of connectivity issues or if the blockchain data isn't updated immediately via the `GitcoinPassportResolver`?
- What is the expected latency for status updates to be reflected in the Passport client?

#### Notes/Assumptions:
- The update in the Passport client should be robust enough to handle a large number of credentials and frequent updates to the on-chain status of credentials. This feature request assumes that users are familiar with the concept of on-chain and off-chain data, and the significance of having credentials on-chain.
- This feature is dependent on the successful implementation of the 'GitcoinPassportResolver' Smart Contract for Passport Attestation Retrieval [#1406](https://github.com/gitcoinco/passport/issues/1406). The on-chain status updates in the Passport client are to be executed via the `GitcoinPassportResolver`.

**Comment by nutrina** [comment]
- the latest docs are here: https://github.com/gitcoinco/eas-proxy/blob/main/docs/01-on-chain-passport-attestation.md#updating-passport
- once the new attestation is implemented and the resolver smart contract is also deployed, that can be used to query for the on-chain passport attestation directly on-chain (no more need for the graphql service)

**Comment by schultztimothy** [comment]
@nutrina and @erichfi  I actually deployed a version of the [Resolver contract](https://goerli.basescan.org/address/0xc0fF118369894100b652b5Bb8dF5A2C3d7b2E343#writeContract) to review after pairing with @aminah-io. I deployed it to [goerli base](https://goerli.basescan.org/) since they have a gnosis safe deployment. We can keep this in blocked until the Resolver ticket is finished and we point to that contract but just wanted to note that it should be working on review.
